consume-more-stuff/server/routes/categories.js

```diff
@@ -1,8 +1,129 @@
 const express = require("express");
 const router = express.Router();
+const Category = require("../db/Models/Category");
+const Item = require("../db/Models/Item");
 
 router.get("/", (req, res) => {
-  res.send(`smoke test - categories`);
+  return Category.fetchAll()
+    .then(cats => {
+      res.json(cats);
+    })
+    .catch(err => console.error(err));
```

Comment:
You'll want to actually return a response here. This will only log the error to console but not send a response back to the browser. Generally want to return a `500` error in the case of a `catch` exception with a general error message (no specifics cuz security)